server/mux.go

```diff
@@ -180,8 +181,19 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 	router.PUT("/chronograf/v1/sources/:id/dbs/:dbid/rps/:rpid", service.UpdateRetentionPolicy)
 	router.DELETE("/chronograf/v1/sources/:id/dbs/:dbid/rps/:rpid", service.DropRetentionPolicy)
 
+	// Set statusFeedURL to a default InfluxData JSON feed if not explicitly set by user
+	statusFeedURL := opts.StatusFeedURL
+	if statusFeedURL == "" {
+		statusFeedURL = "http://news.influxdata.com/feed.json"
```

Comment:
Hey @jaredscheib I think default values should be moved into the CLI options in `server/server.go`